src/node.js

```diff
@@ -623,9 +623,7 @@
         stdout.emit('error', er);
       };
       if (stdout.isTTY) {
-        process.on('SIGWINCH', function() {
-          stdout._refreshSize();
-        });
+        process.on('SIGWINCH', stdout._refreshSize);
```

Comment:
I'm not sure but it looks like this will fail if `_refreshSize` needs to access `this`